cmd/users.go

```diff
@@ -505,6 +523,7 @@ func init() {
 	addUserCmd.Flags().StringVarP(&userFirstName, "firstName", "F", "", "First name of the user")
 	addUserCmd.Flags().StringVarP(&userLastName, "lastName", "L", "", "Last name of the user")
 	addUserCmd.Flags().StringVarP(&userEmail, "email", "E", "", "Email address of the user")
+	addUserCmd.Flags().BoolVarP(&resetPassword, "resetPassword", "", false, "Send a password reset email")
```

Comment:
New flags should use kebab-case. There are some older flags that are camelcase, but the preference is to avoid camelcase where possible.